### DISCUSSION TITLE
Auth.CurrentUser.Id return null

### DISCUSSION THREAD

**Discussion by dimunyo** [supabase/supabase#10448] [Questions]
Hi, Im trying to add supabase on my console app and It returns null even when my username and password is correct
![image](https://user-images.githubusercontent.com/55717823/203197930-8fcbeb02-4285-4b68-ab65-34e9045d584f.png)
![image](https://user-images.githubusercontent.com/55717823/203198056-e597bb9e-96e8-44f9-8b79-2c1b4c87f93d.png)

**Comment by GaryAustin1** [comment]
I don't use c#, but normally you have to await the signIn on js, and seems to show the same in the docs for c#.
